concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-99	99	fighting, troop, army group, battle, quarrel, hubbub, flock	1848	QUARREL
+RonaTas-2011-431-100	100	to add, to join; to buckle up something	1743	JOIN
+RonaTas-2011-431-101	101	clasp		
+RonaTas-2011-431-102	102	measle, smallpox, pocks; skin eruption; decorative embroidery	1054	SMALLPOX
+RonaTas-2011-431-103	103	to bind (something)	1917	TIE
+RonaTas-2011-431-104	104	impurities, tow, chaff	2363	CHAFF (HUSK)
+RonaTas-2011-431-105	105	snail, Helix; snail-shell, pulley, vertebra	1543	SNAIL
+RonaTas-2011-431-106	106	a wine of low quality, a fruit wine		
+RonaTas-2011-431-107	107	secretion, mucus discharged from the eyes		
+RonaTas-2011-431-108	108	squint-eyed, blind in one eye, onearmed, one-legged	87	SQUINT-EYED
+RonaTas-2011-431-109	109	a carpenters axe with a long helve		
+RonaTas-2011-431-110	110	stallion	936	STALLION
+RonaTas-2011-431-111	111	sexual organ of large male animals, of a bull; a whip made from it		
+RonaTas-2011-431-112	112	feast, christening feast (at baptism)		
+RonaTas-2011-431-113	113	to become smaller, remain small, to sway, to totter, tosprain	573	SWAY
+RonaTas-2011-431-114	114	stubborn	3634	STUBBORN
+RonaTas-2011-431-115	115	field-guard; herald; Turkish envoy		
+RonaTas-2011-431-116	116	rough, coarse; ugly; wild, unbridled	1923	ROUGH
+RonaTas-2011-431-117	117	to wind, to turn, to distrort, misinterpret, to spin a yarn	1588	TURN
+RonaTas-2011-431-118	118	bobbin, spindle	277	SPINDLE
+RonaTas-2011-431-119	119	excellent; worthless		
+RonaTas-2011-431-120	120	a short fur coat		
+RonaTas-2011-431-121	121	grist, groats, soft hail		
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
+RonaTas-2011-431-123	123	nut; walnut, Juglans	2042	WALNUT
+RonaTas-2011-431-124	124	pig, Sus domestica	1337	PIG
+RonaTas-2011-431-125	125	to lean, topple over, stream down	3619	LEAN
+RonaTas-2011-431-126	126	to squeeze, to thrust into	1414	SQUEEZE
+RonaTas-2011-431-127	127	church (building and organisation)	1349	CHURCH
+RonaTas-2011-431-128	128	wedge		
+RonaTas-2011-431-129	129	plough	2154	PLOUGH (INSTRUMENT)
+RonaTas-2011-431-130	130	to allow, permit, concede; yield, give way	3238	ALLOW
+RonaTas-2011-431-131	131	to propitiate		
+RonaTas-2011-431-132	132	community work in agriculture, a kind of corvée		
+RonaTas-2011-431-133	133	to arrive, to reach, get to	1387	ARRIVE
+RonaTas-2011-431-134	134	merit		
+RonaTas-2011-431-135	135	morals, morality		
+RonaTas-2011-431-136	136	to tire, lose vitality, slacken, relax	168	REST
+RonaTas-2011-431-137	137	strength, power	2526	POWER
+RonaTas-2011-431-138	138	oath	1712	OATH
+RonaTas-2011-431-139	139	reason, mind	1562	MIND
+RonaTas-2011-431-140	140	river, name of a river	666	RIVER
+RonaTas-2011-431-141	141	pelican, Pelicanus	472	PELICAN
+RonaTas-2011-431-142	142	polecat, Putorius foetidus		
+RonaTas-2011-431-143	143	matchmaker		
+RonaTas-2011-431-144	144	gleaner mouse, Mus spicilegus		
+RonaTas-2011-431-145	145	to abuse, calumniate	1953	ABUSE
+RonaTas-2011-431-146	146	drag-net, a kind of fishing net		
```

Comment:
Are there many different fish nets? I think this could possibly be mapped to `388 FISHNET`

Reply:
I would assume that NETBAG (a net for carrying) and FISHNET are meant. I'd unmap.

Reply:
the original Hungarian word is \<gyalom\> "Zeese" (https://dictzone.com/ungarisch-deutsch-worterbuch/gyalom and https://en.wikipedia.org/wiki/Zeese), so I'll unmap